apps/examples/araplayer/Enumerator.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+typedef enum play_list_type_s {
```

Comment:
Could you please add some detail comment for each of enum? it is very hard to understand...

Reply:
No, please do that this time, I still can't understand whole of code because of that reason.